Docs: Update how to play (#1388)
longturn/freeciv21: docs/Playing/how-to-play.rst

Basic Strategy
==============

First of all, this isn't a *perfect* strategy, but it will get you started playing Freeciv21. Part of the
appeal of Freeciv21 is in developing new strategies as you play more games. This guide is mostly based upon
the Classic ruleset. There are other rulesets out there that ship with the game, but the base information
presented here is the same for all rulesets.

The game is divided into several steps or stages:

#. Choosing the first city site.
#. Selecting your first city locations.
#. Selecting your technological advance.
#. Mapping the countryside.
#. Defending the cities.
#. Deciding which units to build first.
#. Improving the land.
#. Deciding where to build cities.
#. Taking care of the cities.
#. Interacting with other players (diplomacy).
#. Exploring the world.
#. Things to keep in mind.
#. Production phase.
#. Annihilation of your opponents.

Choosing the First City Site
----------------------------

Start the game by wandering around :strong:`briefly` to find a good place to build the first city. Do not feel
tempted to investigate any yellow roofed villages yet, they might contain :unit:`Barbarian` tribes. Build the

> city close to Resources (fish, wheat, wine, or buffalo) and perhaps close to fresh water (River or Lake). Keep

Reviewer: Should the resources in parentheses be capitalized?

Author: Yep, good catch.

in mind that the first city you build will be your capital! It is advised not to place your capital on the
coast. That can open you up to losing it from an attack from the sea.

The idea is to balance the quality of the site you find against getting your first city established as early
as possible. You will find that *balance* becomes a key aspect of playing Freeciv21.

Selecting your First City Locations
-----------------------------------

In the Classic ruleset, you start the game with 5 :unit:`Settlers`, 6 :unit:`Workers`, and 2
:unit:`Explorers`. Once you have planted your first city as your capital, you need to find a place for your
remaining :unit:`Settlers` as quickly as possible. This initial expansion phase of the game is important to

> establishing your base cities. Cities are everything in Freeciv21. The more cities you have the more of every

Reviewer: ```suggestion
establishing your base cities. Cities are everything in Freeciv21. The more cities you have, the more of every
```

> type of resource you can produce. Winning in Freeciv21 is a factor of your nation's ability to out produce

Reviewer: ```suggestion
type of resource you can produce. Winning in Freeciv21 is a factor of your nation's ability to out-produce
```

Could also be "outproduce" as one word, but that seems less clear to me than having it hyphenated

Author: Yea, I like the hyphenated version better too.

your opponents.

> As mentioned in `Choosing the First City Site`_ above, you want to try and place your :unit:`Settlers` so they
> will create cities that have Resources available to them. These tile enhancements give bonuses to help your

Reviewer: ```suggestion
As mentioned in `Choosing the First City Site`_ above, you want to try and place your :unit:`Settlers` so you
can create cities that have Resources available to them. These tile enhancements give bonuses to help your
```

nation grow. In the game help under :title-reference:`Terrain --> Resources` you will find more
information. Another thing you want to look for is sources of fresh water (River and Lake). Placing a city on
or adjacent to sources of fresh water allow you to build a cheap :improvement:`Aqueduct/River` or

> :improvement:`Aqueduct/Lake` city improvements. An :improvement:`Aqueduct` allows your cities to grow larger

Reviewer: The Classic ruleset has neither "Aqueduct/River" nor "Aqueduct/Lake". That's a LTT/LTX thing

Author: Wow, I whiffed on that one. I'll reword.

Author: I'm actually pulling the last part of that paragraph out.

than size 8. If you do not have access to fresh water, you will have to learn the :strong:`Construction`
advance to build the more expensive :improvement:`Aqueduct`.

Selecting your Technological Advance
------------------------------------

With your first city built, your nation will start to produce scientific output called Bulbs. Bulbs are used
to study technological advances. As your nation learns new technologies, new units, city improvements, and
wonders become available to build in your cities.

There are too many scenarios of which technological advance path to take, so make your best judgement. You
will probably want to learn a new form of government. You start the game in Despotism in the Classic ruleset.
:strong:`Monarchy` or :strong:`The Republic` are good early targets. As soon as you have researched a new
government type, start a revolution (
:menuselection:`Civilization --> Government --> Revolution`) and change over to it. Cities operate much better

> as Republics than they do under Despotism, but note that it is much harder to keep military units outside of

Reviewer: ```suggestion
under Republic than they do under Despotism, but note that it is much harder to keep military units outside of
```

:improvement:`Coastal Defense` city improvement later on for defense against an attacking navy. The best
strategy is to build a few of both, but keep in mind that your opponents will find it harder to locate your
city if you do not build it by the sea.

Now is a good time to talk about city planning strategies. There are many strategies to city planning that are
good to discuss here in the early game.

.. note::
  There are a few different kinds of Tilesets available in Freeciv21. For sake of example we are talking about
  the Hex(agonal) type of Tileset in this section. The concepts are very similar with the Square tilesets as
  well. If you are looking at Square tiles, you can change to Hex tiles by going to
  :menuselection:`Game --> Load Another Tileset` and picking ``Hexemplio`` from the list.

Let us first talk about two concepts: Vision Radius and Working Radius. Each city has a vision radius and a
working radius. The vision radius is the tiles that the citizens of a city can see on the map. If an emeny
unit comes within the vision radius of a city, a sentried unit inside of the city will wake up and alert you.

> Within the vision radius is the smaller working radius. When thinking of city planning, you are most concerned

Reviewer: ```suggestion
Within the vision radius is the smaller working radius. When thinking of city planning, you should be most concerned
```

with the working radius of a city. These are the tiles that the citizens of your cities manage to extract

> resources (shields, food, and trade). When the working radius of a city overlaps with that of another city,

Reviewer: Capitalization?

only one city at a time can manage that tile. This is another one of those *balance* items in playing
Freeciv21.

When you have a :unit:`Settler` selected, it will have a red outline shown on the map. This is the city's
working radius if you placed the :unit:`Settler` at that spot with the :menuselection:`Work --> Build City`
command.

> Imagine that this image represents a city with its working radius. The bolded hexagon represents the city

Reviewer: ```suggestion
This image represents a city with its working radius. The bolded hexagon represents the city
```

gold), Entertainers (create happiness), or Scientist (create bulbs). By default, Freeciv21 will add an
Entertainer when you take a citizen off the land.

If you click on the central tile of the land (the "City Center"), the citizens will be rearranged to maximize
food production. You will also want to look at the level of food in the Granary and the amount of surplus food
the city is producing each turn. The city will lose excess food at turn change. For example: The Granary in
the city has 18/20 food (needs two food to grow to next city size) and the city is currently producing +4 food
surplus each turn. This means that at turn change you will lose 2 food as waste at turn change. This is a good
opportunity to move the citizens around to get food to +2 surplus. This could be accomplished by taking a
citizen off a tile producing 2 food and turn it into a Taxman for a turn to get gold. At turn change, open the
city and restore the citizen to farming. This kind of individual city management style is called
"micro-management" and is a very powerful mechanism of game play.

The golden rule of taking care of a city is that there should be at least as many happy citizens as unhappy

> citizens. A city where this is not so falls into disorder. Such cities are labeled with a raised fist or a

Reviewer: ```suggestion
citizens. A city where this is not the case falls into disorder. Such cities are labeled with a raised fist or a
```

Same meaning, simpler wording (I believe). Just an optional suggestion, can be ignored

Author: i agree, it is simpler. good catch

lightning bolt (depending on the tileset). Take care not to let this happen to any of your cities, as cities
in disorder produce nothing, and are prone to revolt.

Interacting with Other Players (Diplomacy)
------------------------------------------

When one of your units first meets a unit of another nation, or finds one of their cities (or equally if
they find you), a basic contact is established between the two nations. This provides each with basic
intelligence about the other, which can be accessed from the
:numref:`Nations and Diplomacy View <Nations and Diplomacy View Button>`.

> This communication will lapse after a ruleset defined number of turns with no contact. Establishing an embassy

Reviewer: I think the exact number of turns for Classic ruleset should be mentioned here

will give a more permanent communication channel, as well as more advanced intelligence such as details of
technology. Embassies are one-way, the nation hosting the embassy receives no benefit and once established,
cannot be revoked.

If you are in contact with another player, then you can arrange a diplomatic meeting. From the

> :numref:`Nations and Diplomacy View <Nations and Diplomacy View Button>` , this is done by selecting the

Reviewer: ```suggestion
:numref:`Nations and Diplomacy View <Nations and Diplomacy View Button>`, this is done by selecting the
```

nation with whom you wish to meet and clicking :guilabel:`Meet`. If the entry under the embassy column is not
blank and the other player is connected (or is a server AI) then a treaty dialog will pop up.

> In this dialog you can negotiate an exchange of assets (maps, vision, advances, cities, or gold), Embassies,

Reviewer: Inconsistent capitalization of "embassy", it isn't capitalized at the beginning of this section, but is capitalized here

or relationship pacts such as a Cease-fire or Peace. The list of items that can be traded through diplomacy is

> ruleset dependent. There is no need to trade like for like; you can trade, say, an advance or city for gold,

Reviewer: Why the semicolon? In other documents we've replaced semicolons with regular sentences

you can consider this buying and selling. Each player builds a list of offered items. To remove an item from
the list, double-click on it. When both players indicate satisfaction, the pact is concluded.

Pacts affect where your units can go and what they can do, and a pact with one nation can affect your
relations with others. under authoritarian governments such as monarchy you can break a pact at any time, but
the representative governments (republic and democracy) have a senate which will block the unprovoked
cancellation of a treaty, unless a foreign diplomat or spy sparks a diplomatic incident. the only way to
dissolve a pact in this situation is to dissolve your government by going into anarchy. the details of pacts

> are described in the diplomacy section.

Reviewer: Capitalization for government names?
